test/useTranslation.loadingLng.spec.js

```diff
@@ -23,14 +23,16 @@ describe('useTranslation loading ns with lng via props', () => {
   afterEach(cleanup);
 
   it('should wait for correct translation with suspense', async () => {
-    const { result, waitForNextUpdate } = renderHook(() =>
-      useTranslation('common', { i18n: newI18n, useSuspense: true, lng: 'de' }),
-    );
-    expect(result.all).toHaveLength(0);
+    const all = [];
+    const { result } = renderHook(() => {
+      const value = useTranslation('common', { i18n: newI18n, useSuspense: true, lng: 'de' });
+      all.push(value);
+      return value;
+    });
+
+    expect(all).toHaveLength(0);
```

Comment:
This expectation isn't doing much here. I opted for leaving it as close to what it was as possible but I think the test could be simplified by removing this assertion as well as the line and the `const all`.